scripts/variantstore/wdl/GvsImportGenomes.wdl

```diff
@@ -292,7 +292,8 @@ task LoadData {
   >>>
   runtime {
       docker: docker
-      memory: "3.75 GB"
+      maxRetries: 3
+      memory: "6 GB"
```

Comment:
Do you have reason to believe we need more memory?  I didn't see this in the 2k samples (and upping the memory will double the cost)